tasks/setup.yaml

```diff
@@ -1,3 +1,7 @@
+variables:
```

Comment:
@MxNxPx Any reason we wouldnt add this to the common tasks repo and then include it here?

Reply:
@zachariahmiller great question.

to do so we need to have [this uds-cli feature](https://github.com/defenseunicorns/uds-cli/issues/414) in order to import the remote task while supporting use of inputs via the `--with` flag.

Reply:
The only alternative I see here is not using the new `with`/`inputs` syntax and just using ENVs...could probably make common tasks work if we did that but not sure which makes more sense?

Reply:
considering that the `--with` flag may be going away, the `--set` flag seems to work, and a common task exists... swapping to import that rather than bake into the uds-core repo